src/app/actions/index.ts

```diff
@@ -1,19 +1,49 @@
-import axios from 'axios';
-import {INFO} from './constants';
+import {CRYPTOS_USER, INFO} from './constants';
+import AsyncStorage from '@react-native-async-storage/async-storage';
 
 const urlApi: string = 'https://data.messari.io/api/v1/assets';
 
 export const getInfo = () => {
   return async function (dispatch) {
     try {
-      const infoApi = await axios.get(urlApi);
+      const infoApi = await fetch(urlApi, {
+        headers: {'x-messari-api-key': '0c9dd049-5e4c-4201-95fe-06ee4317b9ac'},
+      }).then(response => response.json());
 
       return dispatch({
         type: INFO,
-        payload: infoApi.data.data,
+        payload: infoApi.data,
       });
-    } catch (e) {
-      console.log(e);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+};
+
+export const postCryptocurrencyOfUser = (crypto: string) => {
+  return async function () {
+    try {
+      await AsyncStorage.setItem('cryptocurrenciesOfUser', crypto);
```

Comment:
why not create a state for the cryptos that the user uses?

Reply:
I create a state with redux that's call cryptocurrenciesOfUser, the action is getCryptocurrenciesOfUser